docs/testplan/Convergence measurement in data center networks.md

```diff
@@ -0,0 +1,331 @@
+# Convergence measurement in data center networks
+  - [Authors: Kamal Sahu, Keysight Techonologies & 
+              Deepak Singhal, Microsoft Technologies]
```

Comment:
Can these tests be executed on any IXIA platforms. Please update the doc if there are any platform limitations

Reply:
These tests can be run on all IXIA platforms where CP/DP convergence measurement is supported.